app/web/markdown/blog/2025/07/01/releasing-couchers-v1.md

```diff
@@ -0,0 +1,115 @@
+
+That team currently consists of (in alphabetical order) [Aapeli](https://couchers.org/user/aapeli), [Bruna](https://couchers.org/user/bruna), [Charlotte](https://couchers.org/user/karottensaft1), [Chris](https://couchers.org/user/chrisk), [Colleen](https://couchers.org/user/colleen), [Jesse](https://couchers.org/user/jesse), [Marta](https://couchers.org/user/marta), [Mindy](https://couchers.org/user/mindelyn), and [Nicole](https://couchers.org/user/unsettleddown).
+
+## Our Engineering team
+
+We'd also like to say thanks to the engineering team, this year we've had the following wonderful people in our meetings planning and working on the platform: [Aapeli](https://couchers.org/user/aapeli), [Nicole](https://couchers.org/user/unsettleddown), [Chris](https://couchers.org/user/chrisk), [Jesse](https://couchers.org/user/jesse), [Colleen](https://couchers.org/user/colleen), [Laura](https://couchers.org/user/ellebee), [Pablo](https://couchers.org/user/pcolt86), [Shirley](https://couchers.org/user/shirley26), [Rafael](https://couchers.org/user/rafael_ferreira), [Yannic](https://couchers.org/user/spreeni), [Krishna](https://couchers.org/user/krishnag0902), [Caroline](https://couchers.org/user/haoud), [Gracee](https://couchers.org/user/graceegal), Kelvin, Lee, Alena, [Andrei](https://couchers.org/user/andrei_k), [Gabx](https://couchers.org/user/putmyname), Kiki, Navi, Shane, Chalen, [David](https://couchers.org/user/bakeiro), [Ivan](https://couchers.org/user/ivanmen), Jacob, James, [Justyna](https://couchers.org/user/justyna), Kirti and [Rishi](https://couchers.org/user/notatourist).
+
+## Our rehauled branding & marketing strategy
+
+**Our new strategy is centered around a commitment to being the safest, healthiest, and most active couch surfing community.**
+
+From our founding, we have always been defined by our competition: being the non-profit and open-source alternative to CouchSurfing.com™ that takes our community seriuosly and is here to fix the structural problems with other platforms. This has been an incredibly effective strategy, and it has helped us seed the community with the right crowd: a lot of veteran couch surfers with the right spirit.
+
+However, as our community continues to grow further and mature, it's time to stop living in the shadow of another platform and start defining ourselves based on what we actually stand for. This is something the volunteer team and Board have been thinking about a lot, and this has steered our priorities since the start of the year. When you look back at our [original plan](https://couchers.org/plan), it's clear that everything we identified as the issues then were different aspects of community health and safety.
+
```

Comment:
```suggestion
As our community continues to grow and mature, it's time to stop living in the shadow of another platform and start standing in our own identity and values. This is something the volunteer team and Board have been thinking about a lot, and this has steered our priorities since the start of the year. When you look back at our [original plan](https://couchers.org/plan), it's clear that everything we identified as the issues then were different aspects of community health and safety.

```